libkbfs/folder_branch_ops.go

```diff
@@ -6721,24 +6732,13 @@ func (fbo *folderBranchOps) GetUpdateHistory(ctx context.Context,
 }
 
 // GetEditHistory implements the KBFSOps interface for folderBranchOps
-func (fbo *folderBranchOps) GetEditHistory(ctx context.Context,
-	folderBranch FolderBranch) (edits TlfWriterEdits, err error) {
-	fbo.log.CDebugf(ctx, "GetEditHistory")
-	defer func() {
-		fbo.deferLog.CDebugf(ctx, "GetEditHistory done: %+v", err)
-	}()
-
-	if folderBranch != fbo.folderBranch {
-		return nil, WrongOpsError{fbo.folderBranch, folderBranch}
-	}
-
+func (fbo *folderBranchOps) GetEditHistory(
+	_ context.Context, _ FolderBranch) (
+	tlfHistory keybase1.FSFolderEditHistory, err error) {
 	lState := makeFBOLockState()
-	head, err := fbo.getMDForReadNeedIdentify(ctx, lState)
-	if err != nil {
-		return nil, err
-	}
-
-	return fbo.editHistory.GetComplete(ctx, head)
+	md, _ := fbo.getHead(lState)
+	name := md.GetTlfHandle().GetCanonicalName()
+	return fbo.config.UserHistory().GetTlfHistory(name, fbo.id().Type()), nil
```

Comment:
@strib what about here where we use the name from head handle to get TLF history? Would it fail if the name in there was before the SBS?

Reply:
Yes, you're right it would -- though that only affects `/keybase/type/name/.kbfs_edit_history`, not the full user history that will appear in the widget.

If you think it's important, it probably won't be too hard to fix up the names when the handles change, I just didn't feel like complicating things more when I was writing this the first time through.

Reply:
Got it. That's OK; just wanted to make sure this doesn't affect the main path. I'm OK with leaving this as-is. Thanks!